### ISSUE TITLE
Investigation: storage for QuadEdge

### ISSUE THREAD

**Issue by jandam** [gwlucastrig/Tinfour#8]
I want to investigate possibility of custom storage format for QuadEdge. Creating TIN takes some time and loading can take less time.

No action is required. I will try to make some tests based on semi-virtual implementation.

**Comment by gwlucastrig** [comment]
I'm not sure I quite understand what you're working on here...   So I'd like to be sure I understand what you are proposing. 

From time to time, I've thought about the idea of reading and writing the content of a TIN to a data file.  This would be useful in the case where you wanted to re-use the same TIN structure at different times.  Is that what you're trying to do?

I think that you definitely are taking the right approach by starting with the semi-virtual implementation.

**Comment by jandam** [comment]
Yes I want to reuse same TIN structure multiple times. SemiVirtual is indeed good start point. 

Martin JANDA

26. 1. 2017 v 20:29, gwlucastrig <notifications@github.com>:

**Comment by gwlucastrig** [comment]
Martin,
A TIN serialization utility would be an excellent addition to the Tinfour collection.   Let me know if you need anything in terms of support.  Also, if you write some notes on your output file format (a simple text file or word document would be great), I'll take a look at it and let you know if I see any potential issues (I don't expect that there will be any, but just in case...).
